cloudigrade/internal/tests/views/test_dailyconcurrentusageviewset.py

```diff
@@ -0,0 +1,65 @@
+"""Collection of tests for AccountViewSet."""
+import datetime
+
+from django.test import TransactionTestCase
+from django.utils.translation import gettext as _
+from rest_framework.test import APIClient
+
+from api.tests import helper as api_helper
+from api.tests.views.shared.test_dailyconcurrentusageviewset import (
+    SharedDailyConcurrentUsageViewSetTest,
+)
+from util.misc import get_today
+
+
+class InternalDailyConcurrentUsageViewSetTest(
+    SharedDailyConcurrentUsageViewSetTest, TransactionTestCase
+):
```

Comment:
Like with `DailyConcurrentUsageViewSetTest`, since `SharedDailyConcurrentUsageViewSetTest` already extends `TransactionTestCase`, this shouldn't be necessary.

```suggestion
class InternalDailyConcurrentUsageViewSetTest(SharedDailyConcurrentUsageViewSetTest):
```